utils/check_whitespace

```diff
@@ -101,9 +101,20 @@ sub check_whitespace {
 	my $eol;
 	my $nf = 0;
 	my $fstr = decode_file_as_string($full, $file);
+	my $empty = 0;
+	my $is_python = $full =~ /\.py$/;
```

Comment:
I know that this is the fastest way, but I'm not sure if this won't lead to false-negatives when the script doesn't end in .py for whatever reason. I suggest we just look at output of `file $full == "Python script"`.

Reply:
file is not available on Windows...